README.md

```diff
@@ -245,7 +245,7 @@ our discord channels!):
 - [@n1ru4l](https://github.com/n1ru4l)
 - [@acao](https://github.com/acao)
 - [@stonexer](https://github.com/stonexer)
-- [@B2o5T](https://github.com/B2o5T)
+- [@dimaMachina](https://github.com/dimaMachina)
```

Comment:
Looks like this is getting caught by lint-cspell in https://github.com/graphql/graphiql/pull/3848 - i guess we just need to add `dima` to `resources/custom-words.txt`?

Reply:
yep, feel free to send a PR ❤️